packages/remix-fastify/__tests__/index.test.ts

```diff
@@ -132,6 +143,48 @@ describe("fastify createRequestHandler", () => {
         "third=three; Expires=Wed, 21 Oct 2015 07:28:00 GMT; Path=/; HttpOnly; Secure; SameSite=Lax",
       ]);
     });
+
+    it("reloads build on each requests when NODE_ENV=development", async () => {
+      mockedCreateRequestHandler.mockImplementation((build) => async (req) => {
```

Comment:
@cliffordfajardo Thanks for the feedback, There shouldn't be any slowdowns, mainly since the purge modules cache only purges the app's modules and not the whole `node_modules`. This actually works the same as in the `remix` with `express`  projects. 
You can take a look at the code generated when starting a fresh express-remix app (`server.js` file)